### DISCUSSION TITLE
Error: Minified React error #185; visit https://reactjs.org/docs/error-decoder.html?invariant=185 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.

### DISCUSSION THREAD

**Discussion by DayneLinx** [AntlerVC/firetable#1034] [Support / Q&A] [answered]
👉 **Please describe the steps that you took that led to this bug.**
I expanded a column using the handle to make the column wider and the instance crashed.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @DayneLinx, thanks for bringing this to our notice! We're investigating this issue, and we'll get back to you with a solution soon.

**Reply by BeeBombshell** [reply]
Hey @DayneLinx, created an issue for you to track: https://github.com/rowyio/rowy/issues/1111. Therefore, closing this discussion.
